Sources/DynamicNotchKit/DynamicNotch.swift

```diff
@@ -190,18 +191,13 @@ extension DynamicNotch {
             }
         }()
 
-        let panel = NSPanel(
+        let panel = DynamicNotchPanel(
             contentRect: .zero,
             styleMask: [.borderless, .nonactivatingPanel],
             backing: .buffered,
             defer: true
         )
-        panel.hasShadow = false
-        panel.backgroundColor = .clear
-        panel.level = .screenSaver
-        panel.collectionBehavior = .canJoinAllSpaces
         panel.contentView = view
-        panel.orderFrontRegardless()
```

Comment:
Very minor change, but it would make sense to keep `panel.orderFrontRegardless()` in here rather than in the initialization of `DynamicNotchPanel` only for better readability.

Reply:
Reverted, how about the option that moving everything under `DynamicNotchPanel `?
```swift
class DynamicNotchPanel: NSPanel {
    init(contentRect: NSRect, styleMask style: NSWindow.StyleMask, backing backingStoreType: NSWindow.BackingStoreType, defer flag: Bool, contentView: NSView, frame: NSRect) {
        super.init(contentRect: contentRect, styleMask: style, backing: backingStoreType, defer: flag)
        self.hasShadow = false
        self.backgroundColor = .clear
        self.level = .screenSaver
        self.collectionBehavior = .canJoinAllSpaces
        self.contentView = contentView
        self.orderFrontRegardless()
        self.setFrame(frame, display: false)
    }

    override var canBecomeKey: Bool {
        true
    }
}
```

Reply:
No, it would make sense to keep everything that customizes the panel's _functionality_ inside the class, and everything else outside it.